changelogs/CHANGELOG-v0.5.2.md

```diff
@@ -0,0 +1,20 @@
+The bpfman-operator v0.5.2 release is a patch release that contains several
+minor internal updates and is being made primarily to keep in sync with the
+bpfman v0.5.2 release.
+
+## What's Changed
```

Comment:
Should add `Add support for ppc64le and s390x`? No changes were needed in operator in this release, but it needed bpfman support to work.

Reply:
How about I add this in the initial summary blurb?

Reply:
I did both.